docs/newuser.md

```diff
@@ -1,29 +1,41 @@
 # Flow for new users
-This document will go over the flow for getting access to our various services
+This document will go over the flow for getting access to our various services if you are finding difficulties with the specifics of the tasks outlined in this document please see the more detailed [Developer Docs]()
 
-## Publicly Accessible
+## Office 365 Account
+[Office 365](https://portal.office.com), these are provided by emailing DigitalCapabilitiesPMO@homeoffice.gsi.gov.uk, this is not managed by us. Please note that for Single Sign On (SSO) which is using your Office 365 account to log onto other services pop-ups will need to be enabled on your computer.
+
+## VPN
+The VPN is used to be able to get through to our multiple AWS environments as well as other data centers. The profiles can be downloaded from [authd](https://authd.digital.homeoffice.gov.uk) which you can access using your office 365 SSO. The following services can be only accessed when connected to the Dev VPN profile.
+
+## Platform Hub
+
+## Externally Accessible
 These are all services that you can access through normal internet access with personal accounts.
 ### GitHub
-[Github](https://github.com/UKHomeOffice) to access our repositories you must have your personal Github invited to the UK Home Office organisation on Github, someone with access can request this for you.
+[Github](https://github.com/UKHomeOffice) to access our repositories you must have your personal Github invited to the UK Home Office organisation on Github, someone with access can request this for you. Please note that you will have to have your full name as your name on github along with 2 Factor Authentication. If you don't have a personal account on Github then you will need to create one.
```

Comment:
Onboarding on to the GitHub org can now be done via the hub – we probably don't want support requests about this anymore?